analyzer/src/main/kotlin/managers/Bower.kt

```diff
@@ -19,20 +19,237 @@
 
 package com.here.ort.analyzer.managers
 
-import com.here.ort.analyzer.PackageManager
+import ch.frankel.slf4k.info
```

Comment:
We have a policy to always use a wildcard import for this even if only a single log level is used:

    import ch.frankel.slf4k.*